charts/agent-control/templates/installation-job.yaml

```diff
@@ -43,60 +65,77 @@ spec:
           {{- with .Values.installation.extraEnv }}
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          volumeMounts:
+            {{- if .Values.agentControlCd.enabled }}
+            - name: flux-values
+              mountPath: /tmp/flux-values
+              readOnly: true
+            {{- if .Values.agentControlCd.repositoryCertificateSecretReferenceName }}
+            - name: flux-cert-secrets
+              mountPath: /etc/flux-cert
+              readOnly: true
+            {{- end }}
+            {{- end }}
+            {{- with .Values.installation.extraVolumeMounts }}
```

Comment:
in this case we are adding them also if flux is disabled, there is a discrepancy